### PR TITLE
[New Method] Actions/Runner: List enabled repos in organization for Actions

### PR DESCRIPTION
**Pull request by erikkn**
This PR implements the `List selected repositories enabled for GitHub Actions in an organization` method. The endpoint for this feature is `/orgs/{org}/actions/permissions/repositories`. More details around this can be found here: https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-selected-repositories-enabled-for-github-actions-in-an-organization

### CODE REVIEW

**Comment by erikkn** [comment]
@googlebot I signed it!

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:245]:
```suggestion
func (s *ActionsService) ListEnabledReposInOrg(ctx context.Context, owner string, opts *ListOptions) (*ActionsEnabledOnOrgRepos, *Response, error) {
```

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:247]: Please add:

```
	u, err := addOptions(u, opts)
	if err != nil {
		return nil, nil, err
	}
```

before line 247.

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:255]: Please remove lines 253-256 as they are not needed according to the documentation. (Please correct me if I'm wrong... I'm not seeing any mention of them.)

File: github/actions_runners.go
Comment (gmlewis) [inline github/actions_runners.go:258]:
```suggestion
	resp, err := s.client.Do(ctx, req, repos)
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @erikkn - this is a good start.
Please add unit tests to cover this new method.  See our CONTRIBUTING.md guide for full details, including the requirement to run `go generate ...`.

**Comment by erikkn** [comment]
Hi, @gmlewis thank you for your review, really appreciate it :)! Just added your requested changes. Will look into adding the necessary tests now

**Comment by erikkn** [comment]
@gmlewis may I ask you for another review, please :)? Cheers

**Review by gmlewis — APPROVED** [review]
Thank you, @erikkn !
LGTM.

Awaiting second LGTM before merging.

**Comment by erikkn** [comment]
Hey, @wesleimp I know it is still early over there ;) and I obviously don't want to push, but do you have time to review this PR? Would be really great if we can get this in today :).

File: github/actions_runners.go
Comment (nightlark) [inline github/actions_runners.go:21]:
```suggestion
// ActionsEnabledOnOrgRepos represents all the repositories in an organization for which Actions is enabled.
```

File: github/actions_runners_test.go
Comment (nightlark) [inline github/actions_runners_test.go:257]: Is this testing an invalid owner, or bad list options? It looks like opt hasn't changed since the previous (valid) call that was tested. Is the error returned actually related to the arguments passed to the function, or is it an error about not finding the endpoint because the test only has an endpoint for `/orgs/o/actions/permissions/repositories` registered?

Comment (erikkn) [reply github/actions_runners_test.go:257]: Hey @nightlark :)! I am not sure if I understand your comment, could you give me some pointers?

Comment (nightlark) [reply github/actions_runners_test.go:257]: The comment about it testing addOptions doesn't quite feel like it fits what is happening (a newline in the middle of a url where the owner belongs).

I found ~4 other tests with the same thing so I guess that the tests you added were based on one of those. Not sure the comment is accurate in any of those cases, or is really needed in any of those cases (I think throwing a newline in the middle of a url by using \n as the owner or organization argument to any go-github function is likely to give an error, so in a way the test seems kinda pointless -- gmlewis maybe you have some insight I'm missing).

Given that the same sort of test case happens in a few other places, I don't think leaving another instance of it in this test does any harm.

**Review by nightlark — APPROVED** [review]
The two failure scenarios in the test look more generic to me than something that needs to be tested for each endpoint, but where/how to better test or handle those types of invalid inputs seems like a separate discussion.

PR LGTM.

**Comment by erikkn** [comment]
Thank you @nightlark  :)! Is there any chance you can merge the PR for me as well?

File: github/actions_runners_test.go
Comment (gmlewis) [reply github/actions_runners_test.go:257]: Hi @nightlark - yes, this has been added simply to increase the code coverage reports.  Adding in the newline is basically an easy way to generate an error, so it exercises the error-handling path.

**Comment by gmlewis** [comment]
Thank you for the review, @nightlark !
Merging.

**Comment by erikkn** [comment]
@gmlewis thanks for merging! How do you usually deal with new releases :)? Or is there a README I forgot to check about this topic

**Comment by gmlewis** [comment]
Hi @erikkn - releases lately have been on-demand.  I'm assuming you would like one, so I'll work on putting together a release. :smile:

**Comment by erikkn** [comment]
Rockstar! Thank you soo much :100:

**Comment by gmlewis** [comment]
@erikkn - https://github.com/google/go-github/releases/tag/v33.0.0 is now complete.
It is also now accessible here: https://pkg.go.dev/github.com/google/go-github/v33/github

**Pull request merged**
